samples/requirement-targeting-oneway-dependency/nodejs/README.md

```diff
@@ -0,0 +1,227 @@
+---
+page_type: sample
+products:
+- office-365
+languages:
+- javascript
+title: Specify relationships between app capabilities
+description: Microsoft Teams RT sample app in Node.js which specify relationships between app capabilities using (elementRelationshipSet) and hostMustSuppot scenarios.
```

Comment:
As I said in my previous review comment, let's replace this to "Microsoft 365". RT is not just for Teams, but for all of Microsoft 365. Please update all other places where Teams is mentioned in this Readme as well.

Reply:
Also this sentence is confusing to read. Suggest we modify it to be "...which specify one-way-dependency relationships between app capabilities (using "elementRelationshipSet") and functionality requirements of a particular app capability (using "hostMustSupportFunctionalities")..." to be more precise.

Reply:
updated